map-utils.js

```diff
@@ -1,126 +1,159 @@
-EARTH_RADIUS = 6371;
-TILE_SIZE = 256;
+var module = {};
```

Comment:
This works. You can preferably add module to the globals block as well. There are other options such as specifying node as the environment or specifying common.js module format, but what you've done works fine.